src/entity/active_model.rs

```diff
@@ -548,6 +548,11 @@ pub trait ActiveModelTrait: Clone + Debug {
 
         Ok(am)
     }
+
+    /// Returns `true` if any columns were changed.
+    fn is_changed(&self) -> bool {
+        <Self::Entity as EntityTrait>::Column::iter().any(|col| !self.get(col).is_unchanged())
```

Comment:
Hey @kirawi, thanks for contributing! Shouldn't this be...?

```suggestion
        <Self::Entity as EntityTrait>::Column::iter().any(|col| self.get(col).is_unchanged())
```

Reply:
i think it is good with !, as on column we can only check if it is not changed and on active model we will check if it is changed

Reply:
If symmetry is desired, I could change it to be `is_unchanged`

Reply:
for me `is_changed` is way better then `is_unchanged`

Reply:
Oppps... sorry, please ignore this. I thought it's `is_changed` but in fact it's `is_unchanged`

Reply:
😄